README.rst

```diff
@@ -62,6 +62,9 @@ Note that "unused" dependencies should be taken with a grain of salt, package
 may or may not still need it in some other way than ``DT_NEEDED``. ``dlopen()``,
 build time checks/headers, executables, and other potential non-library files.
 
+Note that installed packages' files are considered as candidates only.
+Note that there won't be a warning if a library isn't provided by any package.
```

Comment:
For the sake of not having too much redundant information, I did commit 80ddaa8aac3290051de86d5c8fb6fbd504ebdea7 -- so I'd skip this.

Reply:
Then we could remove the previous paragraph, either :smile: (a joke)